src/Sylius/Behat/Context/Api/EmailContext.php

```diff
@@ -169,6 +169,14 @@ public function recipientShouldReceiveNoEmails(string $recipient): void
         Assert::false($this->emailChecker->hasRecipient($recipient));
     }
 
+    /**
+     * @Then /^only one email should have been sent to "([^"]*)"$/
```

Comment:
do we need to use regex here?
IIRC by default we use:
```suggestion
     * @Then only one email should have been sent to :recipientEmail
```

Reply:
commend valid alos to next 2 files